Minebot/src/main/java/net/famzangl/minecraft/minebot/ai/task/AITask.java

```diff
@@ -97,7 +98,7 @@ protected int computeGameTickTimeout(AIHelper helper) {
 		if (applyToDelta(world)) {
 			return (int) (world.getTimeInTicks() * 1.3f);
 		} else {
-			return 5 * 20;
+			return 6 * 20;
```

Comment:
You should not reach this branch when mining - applyToDelta should be reached here and will calculate the time the block needs to be broken (e.g. obsidian takes a lot more time)

Reply:
applyToDelta seems to get it wrong because for some reason mining ancient debris from the underside takes longer

Reply:
That's interesting. I don't see a mention for this in the Minecraft wiki, so it is probably the same mining time.

What might happen here is that applyToDelta for the task that moves from bottom to top does not work correctly. 

I found and fixed the source problem (did not work for obsidian and other materials as well):
https://github.com/michaelzangl/minebot/commit/4b996b19ed1482714ea3c9102130354637346ca3

Reply:
> Airbridge and fill still don't work for me (and haven't), not too sure

They work now, see comment with the wrong bounds of air.

Reply:
Wow nice :D